hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSImage.java

```diff
@@ -587,7 +587,7 @@ void doImportCheckpoint(FSNamesystem target) throws IOException {
     
     if (checkpointEditsDirs == null || checkpointEditsDirs.isEmpty()) {
       throw new IOException("Cannot import image from a checkpoint. "
-                            + "\"dfs.namenode.checkpoint.dir\" is not set." );
+                            + "\"dfs.namenode.checkpoint.edits.dir\" is not set." );
```

Comment:
Looks like there are some checktyles that need fixing. Maybe you can configure `checkstyle.xml` in your IDE. See `./hadoop-github/hadoop-build-tools/src/main/resources/checkstyle/checkstyle.xml`. The other changes look good to me.

Reply:
great! Thanks @tomscut for your advice,